haiku-apps/xrs/xrs-1.2.recipe

```diff
@@ -0,0 +1,49 @@
+SUMMARY="A simple rhythm station for creating loops"
+DESCRIPTION="XRS is a useful rhythm station for creating your own \
+looping music and could be used to create repeating background music \
+for videos or even a ringtone for your phone."
+HOMEPAGE="https://github.com/HaikuArchives/XRS"
+COPYRIGHT="2001-2003 Giuseppe Gargaro & Andrea Anzani"
+LICENSE="MIT"
+REVISION="1"
+srcGitRev="d91e2b8b808ace64b3032a75728b9fa3c43760f0"
+SOURCE_URI="https://github.com/HaikuArchives/xrs/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256="11c656e544e59f345296f03f7c9ba513365f699cdb51608591a9e9e8a9dcbe6b"
+SOURCE_FILENAME="XRS-$portVersion-$srcGitRev.tar.gz"
+SOURCE_DIR="XRS-$srcGitRev"
```

Comment:
```suggestion
SOURCE_DIR="XRS-$srcGitRev/XrsPalm"
```

If only that folder is needed for build, you could specify `SOURCE_DIR` like above to avoid `cd`.